zha/units.py

```diff
@@ -157,3 +157,6 @@ class UnitOfEnergy(StrEnum):
 
 # Percentage units
 PERCENTAGE: Final[str] = "%"
+
+# Electrical conductivity units
+ELECTRICAL_CONDUCTIVITY_DECISIEMENS_PER_METER: Final[str] = "dS/m"
```

Comment:
Also, `dS/m` is not a valid unit for `SensorDeviceClass.CONDUCTIVITY` in Home Assistant.

Like correctly copied in the `sensor/const.py` file, it has to be `S/cm`, `mS/cm`, or `µS/cm`.
If, according to ZCL, another unit is on that attribute by default, we need to convert it first. Might need to set `_divisor = 10` for example then.

But in this case, I think we can just use `mS/cm`. That's the same as `dS/m`. So, use `UnitOfConductivity.MILLISIEMENS_PER_CM` as the unit for the entity then.

Reply:
Turns out `µS/cm` seems to be the unit commonly used for soil electrical conductivity.

Reply:
Ah yes, according to ZCL R8 4.10.2.1.1.1, line 8625 (page 362), `measured_value * 10` is in `mS/m`.
Since we're showing the value "raw" (with no divider), the unit should indeed be `µS/cm`.